resources/timezones/README.md

```diff
@@ -0,0 +1,40 @@
+# Timezone Mapper
+
+The timezone mapper allows one to get a likely timezone for a particular phone
+number. The timezone returned is the canonical ID from [CLDR](
```

Comment:
Optional: Perhaps:
... allows a likely timezone to be obtained for a given phone number.